charts/cray-certmanager/templates/cert-manager/configmap.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cert-manager
+data:
+  modify_securityContext.sh: |-
+    #!/bin/sh
+
+    echo "Reconfiguring cray-certmanager-cert-manager deployment"
```

Comment:
You should be able to replace all of this with the kubectl patch command, see the documentation : https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/